voter-dapp/src/ActiveRequests.js

```diff
@@ -13,13 +13,13 @@ import Typography from "@material-ui/core/Typography";
 import { formatDate, formatWei } from "./common/FormattingUtils.js";
 import { VotePhasesEnum } from "./common/Enums.js";
 import { decryptMessage, deriveKeyPairFromSignatureMetamask, encryptMessage } from "./common/Crypto.js";
+import { useTableStyles } from "./Styles.js";
 const { getKeyGenMessage } = require("./common/EncryptionHelper.js");
 
 const editStateReducer = (state, action) => {
-  console.log("::editStateReducer()");
   switch (action.type) {
     case "EDIT_COMMIT":
-      return { ...state, [action.index]: action.price };
+      return { ...state, [action.index]: action.price ? action.price : "0" };
```

Comment:
Just for my information - what was wrong with this before?

Reply:
With the reducer refactoring, I eliminated the separate state variable for tracking whether the commit is in edit mode or not, and introduced this bug in the process.